hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ReconTaskControllerImpl.java

```diff
@@ -219,7 +220,9 @@ public ReconTaskStatusDao getReconTaskStatusDao() {
   @Override
   public synchronized void start() {
     LOG.info("Starting Recon Task Controller.");
-    executorService = Executors.newFixedThreadPool(threadCount);
+    executorService = Executors.newFixedThreadPool(threadCount,
+        new ThreadFactoryBuilder().setNameFormat("Recon Task Thread - %d")
```

Comment:
Nit: avoid space in thread name to make it easier to cut columns from log by whitespace.

```suggestion
        new ThreadFactoryBuilder().setNameFormat("ReconTaskThread-%d")
```